docker-compose.yml

```diff
@@ -70,13 +83,41 @@ services:
     env_file: ./docker.env
     environment:
       - SERVICE_TYPE=DB_SSH_CONNECTOR_SERVICE
+      - DBCONNECTOR_POSTGRES_POOL_MAX_SIZE=100
+      - DBCONNECTOR_QUERY_TIMEOUT_MS=120000
     networks:
       - db-ssh-connector-network
     volumes:
       - ssh:/retool_backend/autogen_ssh_keys
       - ./keys:/retool_backend/keys
     restart: on-failure
 
+  workflows-worker:
```

Comment:
this won't crash if using version 3.6+ onwards

don't think it warrants any more warning given we are now 3.10+